frontend/sass/styles.scss

```diff
@@ -1,3 +1,5 @@
+@import '../../node_modules/uswds/dist/css/uswds.css';
```

Comment:
having the hardcoded path to our project root `node_modules` seems clunky, but I don't see an easy way to avoid it if we want to bring the uswds into sass-land.